package.json

```diff
@@ -16,7 +16,8 @@
   "gypfile": true,
   "scripts": {
     "install": "node-gyp rebuild",
-    "pretest": "jscs . && jshint .",
+    "instrument": "node node_modules/tap/node_modules/nyc/node_modules/istanbul/lib/cli.js instrument --output backend.cov --no-compact --preserve-comments backend",
+    "pretest": "jscs . && jshint . && npm run instrument",
     "test": "tap test/lab/*.test.js test/*.test.js"
```

Comment:
To actually generate coverage on CI, either the `test` script needs to be modified to include `--coverage --coverage-report=cobertura` or build-framework needs to be modified it inject it when `tap@1.x` is detected.